Sources/ExyteChat/Views/Attachments/FullscreenMediaPages.swift

```diff
@@ -56,7 +56,9 @@ struct FullscreenMediaPages: View {
                     if viewModel.showMinis {
                         ScrollView(.horizontal) {
                             HStack(spacing: 2) {
-                                ForEach(viewModel.attachments.enumerated().map({ $0 }), id: \.offset) { (index, attachment) in
+                                ForEach(viewModel.attachments
+                                    .filter { $0.fullUploadStatus == nil || $0.fullUploadStatus == .complete  }
```

Comment:
if the client is not using upload status functionality (nil) then show the attachment in full screen media otherwise only show completed uploads